### ISSUE TITLE
How to calculate the loss for pspnet?

### ISSUE THREAD

**Issue by UcanSee** [meetshah1995/pytorch-semseg#195]
I want to train pspnet on my own dataset, but I don't know how to calculate the loss because there are two outputs x and x_aux when training, the x_aux is not same size with the input, how should i  calculate the loss?
